### PR TITLE
Update the "all SDK" page to have source and docs

### PR DESCRIPTION
**Pull request by v-xuto**
Contributes to https://github.com/Azure/azure-sdk/issues/1538, manually update the "non New" SDKs with links to source and docs pulled from SDK Inventory.
1. For Source url,  if there is no source code of the corresponding version, we directly add the source code of the corresponding package under the master branch in the CSV file.
2. For MSDocs, when msdocs_url does not match the template msdocs_url_template, we manually add the corresponding link in the CSV file.

@jongio  for notification.

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -28,58 +28,58 @@
...
 "Azure.ResourceManager.Storage","","1.0.0-preview.1","Resource Management - Storage","Storage","storage","","","mgmt","",""
-"Azure.AI.InkRecognizer","","1.0.0-preview.1","Azure.AI.InkRecognizer","","NA","NA","NA","","",""
-"Azure.Analytics.Synapse.AccessControl","","1.0.0-preview.1","Azure.Analytics.Synapse.AccessControl","","NA","NA","NA","","",""
-"Azure.Analytics.Synapse.Artifacts","","1.0.0-preview.1","Azure.Analytics.Synapse.Artifacts","","NA","NA","NA","","",""
-"Azure.Analytics.Synapse.Spark","","1.0.0-preview.1","Azure.Analytics.Synapse.Spark","","NA","NA","NA","","",""
-"Azure.AspNetCore.DataProtection.Blobs","","1.0.0-preview.1","Azure.AspNetCore.DataProtection.Blobs","","NA","NA","NA","","",""
-"Azure.AspNetCore.DataProtection.Keys","","1.0.0-preview.1","Azure.AspNetCore.DataProtection.Keys","","NA","NA","NA","","",""
+"Azure.AI.InkRecognizer","","1.0.0-preview.1","Azure.AI.InkRecognizer","Ink Recognizer","https://github.com/Azure/azure-sdk-for-net/tree/Azure.AI.InkRecognizer_1.0.0-preview.1/sdk/cognitiveservices/InkRecognizer","NA","NA","","",""
```
Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:31]: @weshaggard Can answer more definitively, but I believe if you set RepoPath to cognitiveservices/InkRecognizer it should automattically format appropriately.

**Comment by jongio** [comment]
@weshaggard - I looked at the CI logs and I don't think link checker ran on all these new links.  Can you investigate?  I definitely want to verify the links before merging.

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:31]: This package should have hide=true as it is a package we removed from the repo and don't plan to ship any more.

File: _includes/releases/pkgrow.md [hunk 2]
```diff
@@ -53,6 +56,8 @@
   <td>
     {% if item.RepoPath == "NA" or item.RepoPath contains "http" %}
         {% assign source_url = item.RepoPath %}
+    {% elsif item.DisplayName contains "@azure/arm" or item.DisplayName contains "@azure/cognitiveservices" %}
```
Comment (weshaggard) [inline _includes/releases/pkgrow.md:59]: I don't want to add specific hard-coded examples in our templates. Can instead just add the full url

File: _includes/releases/pkgrow.md [hunk 3]
```diff
@@ -27,10 +27,13 @@
     {% endif %}
 
     {% if item.VersionPreview != "" %}
-        {% if item.MSDocs != "NA" %}
+        {% if item.MSDocs == "" %}
```
Comment (weshaggard) [inline _includes/releases/pkgrow.md:30]: I think this condition still needs to account for "NA" which means it doesn't have a link.

**Comment by weshaggard** [comment]
>@weshaggard - I looked at the CI logs and I don't think link checker ran on all these new links. Can you investigate? I definitely want to verify the links before merging.

As far as I can tell it ran on all the links. However there is a limitation which we do get 429 rate-limiting status for a number of sites like github and so in those cases it isn't clear if github will always return a 429 even if something doesn't exist after we hit the rate-limited threshold or if it will return 404. I suspect it blindly returns 429 once you hit the limit and unfortunately we don't have a great way to defend against that in an automated way. I suggest @v-xuto to run the verify-link.ps1 script locally against each language page independently to help verify them..

**Comment by jongio** [comment]
@tzhanl / @v-xuto - Can you please provide a report to show that all of the links have been tested?  Thanks, Jon

**Comment by v-xuto** [comment]
@jongio  - The following  test.log to show that all of the links have been tested.

[test.log](https://github.com/Azure/azure-sdk/files/5089146/test.log)

**Comment by jongio** [comment]
Thank you @v-xuto - @weshaggard - Would be great to get this merged.  Thanks, Jon

**Comment by weshaggard** [comment]
@v-xuto you will need to resolve conflicts before we can merge this change. 

Also for my information what verify_links.py script are you using to test the links?

**Pull request merged**